docs/reference/commandline/create.md

```diff
@@ -68,7 +68,11 @@ Creates a new container.
       -u, --user=""                 Username or UID
       --ulimit=[]                   Ulimit options
       --uts=""                      UTS namespace to use
-      -v, --volume=[]               Bind mount a volume
+      -v, --volume=[]               Bind mount a volume with: [host-dir:]container-dir[:<options>], where
```

Comment:
hi @thaJeztah  I added it like this

The 'host-dir' can either be an absolute path or a name value.

It is not needed to edit with [host-dir | volume-name:], as this link below
https://github.com/docker/docker/blob/master/docs/reference/run.md#volume-shared-filesystems
talked about this, we'd better keep consistent style.  Thanks